src/Reflection/BetterReflection/SourceLocator/ComposerJsonAndInstalledJsonSourceLocatorMaker.php

```diff
@@ -111,19 +109,15 @@ public function create(string $projectInstallationPath): ?SourceLocator
 			)),
 		);
 
+		$classMapDirectories = array_filter($classMapPaths, 'is_dir');
 		foreach ($classMapDirectories as $classMapDirectory) {
-			if (!is_dir($classMapDirectory)) {
-				continue;
-			}
 			$locators[] = $this->optimizedDirectorySourceLocatorRepository->getOrCreate($classMapDirectory);
 		}
 
 		$files = [];
 
+		$classMapFiles = array_filter($classMapPaths, 'is_file');
 		foreach (array_merge($classMapFiles, $filePaths) as $file) {
-			if (!is_file($file)) {
-				continue;
```

Comment:
What about checking `is_file` for `$filePaths` here?

Reply:
good catch, fixed.

Reply:
How is this change different to what already happened to previous version of the code?

Reply:
oh I see. I read your initial comment differently.

just reworked the code to get rid of `array_filter()`.